index.bs

```diff
@@ -501,9 +502,9 @@ Element</h4>
 which is initialized to an empty list.
 This list holds <dfn interface>IntersectionObserverRegistration</dfn> records,
 which have an <dfn attribute for=IntersectionObserverRegistration>observer</dfn> property
-holding an {{IntersectionObserver}}
-and a <dfn attribute for=IntersectionObserverRegistration>previousThresholdIndex</dfn> property
-holding a number between -1 and the length of the observer's {{IntersectionObserver/thresholds}} property (inclusive).
+holding an {{IntersectionObserver}}, a <dfn attribute for=IntersectionObserverRegistration>previousThresholdIndex</dfn> property
+holding a number between -1 and the length of the observer's {{IntersectionObserver/thresholds}} property (inclusive), and
+a <dfn attribute for=IntersectionObserverRegistration>previousIsIntersecting</dfn> property holding a boolean or <code>undefined</code>.
```

Comment:
Can we get rid of the "undefined" value for previousIsIntersecting, and just initialize it to "false"?  The -1 value for previousThresholdIndex will still serve the purpose of forcing the first notification, so it seems like we can safely make previousIsIntersecting and strict boolean.

Reply:
Good catch! Yes, we can keep it strict boolean.